DOCS-9331 add LDAP note (#1567)
mulesoft/docs-connectors: ldap/3.2/modules/ROOT/pages/index.adoc

Release Notes: xref:release-notes::connector/ldap-connector-release-notes-mule-4.adoc[LDAP Connector Release Notes] +
Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-ldap-connector/[LDAP Connector]

[NOTE]

> LDAPS (LDAP over SSL) is currently not supported. This version supports only LDAP over TLS.

Reviewer: ```suggestion
This version of the connector supports only LDAP over TLS.
```
